http://cvs.sourceforge.net/viewcvs.py/gaim/gaim/src/dialogs.c?r1=1.594&r2=1.595&diff_format=u
Fixes smileys in away message dialog

===================================================================
RCS file: /cvsroot/gaim/gaim/src/dialogs.c,v
retrieving revision 1.594
retrieving revision 1.595
diff -u -r1.594 -r1.595
--- gaim/gaim/src/dialogs.c	2004/04/24 01:43:11	1.594
+++ gaim/gaim/src/dialogs.c	2004/04/24 20:44:41	1.595
@@ -815,6 +815,8 @@
 	if (gaim_prefs_get_bool("/gaim/gtk/conversations/spellcheck"))
 		gaim_gtk_setup_gtkspell(GTK_TEXT_VIEW(ca->text));
 	gtk_imhtmltoolbar_attach(GTK_IMHTMLTOOLBAR(ca->toolbar), ca->text);
+	gtk_imhtmltoolbar_associate_smileys(GTK_IMHTMLTOOLBAR(ca->toolbar), "default");
+	gaim_setup_imhtml(ca->text);
 
 	gtk_container_add(GTK_CONTAINER(sw), ca->text);
 
